### DISCUSSION TITLE
Detecting window drag over another window

### DISCUSSION THREAD

**Discussion by smallB007** [lapce/floem#585] [Q&A] [answered]
Hi, how can I detect window being dragged over another window? Both windows created with floem.
Thank you.

**Comment by smallB007** [comment]
Guys, could you please answer that? Is it possible or not?
Thank you

**Comment by dzhou121** [comment, marked as answer]
Because the events are provided by winit and I can't see such events emited by winit, so that should not be directly possible. 

But you should be able to record all the window positions and sizes, and do some checks yourself?

**Reply by smallB007** [reply]
Thank you for your reply.
